tools/compose-external-boot-image-yml.sh

```diff
@@ -3,7 +3,16 @@
 set -e
 
 yq() {
-  docker run -i --rm -v "${PWD}/":/workdir intoiter/yq:3.1.0 -y -i "$@"
+  docker run -i --rm -v "${PWD}/":/workdir -w /workdir mikefarah/yq:4.40.5 "$@"
```

Comment:
Is this part of the PR? Or just a convenient place to update yq?

Reply:
convenient to update yq:

```
It starts to use yq from the same image used by tools/compose-image-yml.sh avoiding download another yq image and it fixes the generation of the parsed file (build.yml).
```